transcom/mymove
# B 19830 int revised

## [B-19830](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19830)

## Summary

> [!IMPORTANT]  
> This is the second PR regarding this story. The original can be found [here](https://github.com/transcom/mymove/pull/12784). Please consult the update comment and revised AC on Agility.

There was an issue with this task that came to light during PO Acceptance for this story. Originally, it was understood that the subtitle text needed to be changed, but this is incorrect. The original subtitle text will be restored, and instead the new text will read under the Name field. The reason for this follows:

Currently, the Trusted Agents section is automatically populated by the application with mto agents attached to the move associated with a PPM. However, mto agents currently only consists of releasing and receiving agents, and trusted agents are a different role entirely and should not be automatically configured by the application. Instead, they should be configured manually by the releasing office only. 

Therefore, this story now needs to remove the existing automatic population of MTO Agents and replace their information with the specified text "Trusted Agent Requires POA or Letter of Authorization".

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the customer application, create and submit a PPM with an AOA Request
2. Access the office application, login as a Service Counselor, approve the AOA Request, orders, and shipment
3. Access the customer application again as the previous user, return to your previous move
4. Download the AOA Packet
5. Ensure that the Trusted Agent section has no data filled except for the copy that reads "Trusted Agent Requires POA 
or Letter of Authorization" and The Trusted Agent subtitle is "Allowed to act on customers behalf via letter of authorization". 
<img width="634" alt="image" src="https://github.com/transcom/mymove/assets/148906585/68293680-f6c6-4dea-9c01-6eda8227cf97">

6. Optionally, repeat for generating the Payment Packet.

### Backend
